ansible/roles/commcarehq/templates/logrotate.conf.j2

```diff
@@ -0,0 +1,29 @@
+# see "man logrotate" for details
+# rotate log files weekly
+weekly
+
+# packages drop log rotation information into this directory
+include /etc/logrotate.d
+
+# no packages own wtmp, or btmp -- we'll rotate them here
+/var/log/wtmp {
+    missingok
+    monthly
+    create 0664 root utmp
+    rotate 1
+}
+
+/var/log/btmp {
+    missingok
+    monthly
+    create 0660 root utmp
+    rotate 1
+}
+
+{{ log_home }}/*.log {
+    weekly
+    missingok
+    compress
+    delaycompress # compress on second rotation
+    olddir {{ rotated_log_home }}
```

Comment:
OK sounds good. Before applying this to anywhere that currently has log
rotation set up though, you'll need to (1) disable whatever logrotation is
currently set up and (2) move all the current rotated logs into the new
directory.

You might also want to look into what configuration we use on hqproxy0.

Cheers,
Danny

On Fri, May 8, 2015 at 4:02 PM, Ben <notifications@github.com> wrote:

> In ansible/roles/commcarehq/templates/logrotate.conf.j2
> <https://github.com/dimagi/commcarehq-ansible/pull/100#discussion_r29971776>
> :
>
> > +    rotate 1
> > +}
> > +
> > +/var/log/btmp {
> > +    missingok
> > +    monthly
> > +    create 0660 root utmp
> > +    rotate 1
> > +}
> > +
> > +{{ log_home }}/*.log {
> > +    weekly
> > +    missingok
> > +    compress
> > +    delaycompress # compress on second rotation
> > +    olddir {{ rotated_log_home }}
>
> ah was just about to mention the that they were gzipped. zgrep sounds
> cool. i did do it on purpose because i've seen other places that are
> littered with log rotates that it's hard to find what real files are there
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcarehq-ansible/pull/100/files#r29971776>.
>